internal/commands/push/command_test.go

```diff
@@ -1029,11 +1029,170 @@ Try instead: realm-cli push --local testdata/dependencies --remote appID --inclu
 	})
 }
 
+func TestPushHandlerUseCreateNewApp(t *testing.T) {
```

Comment:
this func name could be a bit more descriptive for what we're testing -- pushing from nested dirs.
I like nick's previous suggestion `TestPushCommandCreateNewAppNested`

Reply:
Pushing into a nested directory doesn't seem entirely descriptive to me either... This test feels more like an integration test verifying that the handler calls create new app correctly (i.e. writes to the config file as expected).

Reply:
I wouldn't go too crazy on names here... the only way these functions are "called" is through `go test...` and for my own laziness, I like keeping the names hierarchical so my `go test` regexes are easy/simple to write (read: putting that "Use" in the middle of the name makes it harder for me to potentially run _just_ the tests related to `CreateNewApp`)

I'd agree `...CreateNewAppNested` sounds funny... but again, for ease of regex-ing it allows for meaningful suites of tests to be run via:
`-run 'TestPushCommand'`
`-run 'TestPushCommandCreateNewApp'`
`-run 'TestPushCommandCreateNewAppNested'`

that middle one is :neat: b/c like you said, this is more or less a test that makes sure `Handler` is invoking `createNewApp` the proper way... so we'd still be able to test all things related to `createNewApp` that way

Reply:
Using `TestPushHandlerCreateNewApp` since this is verifying the integration of handler and create new app at a high level.